cisco_aci/tests/test_cisco.py

```diff
@@ -75,3 +76,20 @@ def test_recover_from_expired_token(aggregator, case, api_kwargs):
     # Assert cookie to check the session changed
     assert get_calls[0].kwargs['headers']['Cookie'] == 'cookie'
     assert get_calls[1].kwargs['headers']['Cookie'] != 'cookie'
+
+
+@pytest.mark.parametrize(
+    'extra_config, expected_http_kwargs',
+    [
+        pytest.param({}, {'auth': (common.USERNAME, common.PASSWORD), 'verify': True}, id='new auth config'),
+        pytest.param({'ssl_verify': True}, {'verify': True}, id='legacy ssl verify config True'),
+        pytest.param({'ssl_verify': False}, {'verify': False}, id='legacy ssl verify config False'),
+    ],
+)
+def test_config(aggregator, extra_config, expected_http_kwargs):
+    instance = deepcopy(common.CONFIG_WITH_TAGS)
+    instance.update(extra_config)
+    check = CiscoACICheck(common.CHECK_NAME, {}, [instance])
+
+    for key in expected_http_kwargs:
+        assert check.http.options[key] == expected_http_kwargs[key]
```

Comment:
```suggestion
    actual_options = {k:v for k,v in check.http.options if k in expected_http_kwargs}
    assert expected_http_kwargs == actual_options
```

(nit) Asserting all key/value pairs will give a better/complete output in case of test failure.